app/src/NativeBase/Containers/MyExperienceContainer.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * @file the MyExperience screen for the STA Volunteer App
+ */
+import {
+  Heading,
+  VStack,
+  HStack,
+  Checkbox,
+  ScrollView,
+  Flex,
+  Button,
+  Box,
+  View,
+} from 'native-base'
+import StaRibbon from '../Components/StaRibbon'
+import StaThemeLogo from '@/NativeBase/Assets/Images/Logos/sta-ribbon-logo.svg'
+import FreeSearchBar from '../Components/FreeSearchBar'
+import React from 'react'
+import { RoleGroup, roleGroups } from '@/Services/modules/projects/roleGroups'
+
+const MyExperienceContainer = () => {
+  const [skillsValue, setSkillsValue] = React.useState<string[]>([])
+  const [searchTxt, setSearchTxt] = React.useState<string>('') // search text
+
+  /**
+   * Filter the role groups based on the search text entered by the user
+   */
+  const filteredSkills = roleGroups.filter((roleGroup: RoleGroup) => {
```

Comment:
Another optional bit of shorthand you could use here -- this function could be simplified to this (because it's a one-line function we don't have to use curly brackets and `return`):

```
const filteredSkills = roleGroups.filter((roleGroup: RoleGroup) =>
  roleGroup.groupName.toLowerCase().includes(searchTxt.toLowerCase()),
)
```